roles/faucet/templates/config.json.j2

```diff
@@ -0,0 +1,31 @@
+{
+    "GLOBAL_RL": {
+        "ID": "GLOBAL",
+        "RATELIMIT": {
+            "REVERSE_PROXIES": 2,
+            "MAX_LIMIT": 40,
+            "WINDOW_SIZE": 1,
+            "PATH": "/",
+            "SKIP_FAILED_REQUESTS": false
+        }
+    },
+    "evmchains": [
```

Comment:
If we use the same property names, I guess we can just capitalize them here. This will be less painful to maintain.